website/server/libs/cron.js

```diff
@@ -130,6 +130,14 @@ function trackCronAnalytics (analytics, user, _progress, options) {
 
 function awardLoginIncentives (user) {
   if (user.loginIncentives > 50) return;
+
+  //  Remove old noitification if it exists
+  user.notifications
+    .toObject()
+    .find((notif, index) => {
```

Comment:
I want it to remove multiple notifications, not the first. So, I don't want to short circuit.